agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v1.yaml

```diff
@@ -114,6 +117,9 @@ spec:
         #     new_k8s: true
         #     k8s_cluster_name: production
         volumeMounts:
+        - mountPath: /etc/modprobe.d
+          name: modprobe-d
+          readOnly: true
```

Comment:
This ought to be for the `sysdig-agent-kmodule` init container, not for `sysdig-agent`

Reply:
This has been fixed in latest commit